File: 251003/Stasevichy/discussion/src/main/java/com/lab/labDC/entity/Notice.java
package com.lab.labDC.entity;

import lombok.*;
import org.springframework.data.cassandra.core.mapping.CassandraType;
import org.springframework.data.cassandra.core.mapping.Table;
import org.springframework.data.annotation.Id;

@Table("tbl_notice")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class Notice {

    @Id
    @CassandraType(type = CassandraType.Name.UUID)
    private Long id;

    @CassandraType(type = CassandraType.Name.TEXT)
    private String content;

    @CassandraType(type = CassandraType.Name.BIGINT)
    private Long storyId;

}

File: 251001/Voytov/discussion/src/main/kotlin/org/ex/discussion/model/Notice.kt
package org.ex.discussion.model

import lombok.Getter
import lombok.Setter
import org.springframework.data.cassandra.core.mapping.Column
import org.springframework.data.cassandra.core.mapping.PrimaryKey
import org.springframework.data.cassandra.core.mapping.Table

@Setter
@Getter
@Table("notice")
data class Notice (

    @PrimaryKey
    @Column("id")
    val id: Long,

    @Column("country")
    val country: String,

    @Column("newsId")
    var newsId: Long,

    @Column("content")
    var content: String
)